visualizer/style.css

```diff
@@ -46,7 +46,8 @@ html {
   --footer-bg-color: rgb(15, 18, 26);
   --footer-color: rgba(255, 255, 255, 0.9);
 
-  --options-menu-bg-color: rgba(var(--banner-bg-color-val), 0.9);
+  --options-menu-bg-color: rgb(0, 0, 0);
+  --options-menu-bg-hover: rgb(10, 10, 10);
```

Comment:
For consistency it'd make sense to use this for the tooltip and (non-disabled) button backgrounds on hover as well; and give it a more general name (maybe --clickable-bg-hover).

And then I think we wouldn't need --options-menu-bg-color with the above change